devfile.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: 1.0.0
+metadata:
+  generateName: che-devfile-registry-
+
+components:
+  - alias: che-devfile-registry-prod
+    image: 'quay.io/eclipse/che-devfile-registry:nightly'
+    mountSources: true
+    endpoints:
+      - name: http
+        port: 8080
+    command:
+      - sleep
+      - infinity
+    memoryLimit: 512Mi
+    type: dockerimage
+    volumes:
+      - name: images
+        containerPath: /usr/local/apache2/htdocs/images
+
+  - alias: dev
+    image: 'quay.io/eclipse/che-plugin-registry-dev:latest'
```

Comment:
what is the use of this image there ?
we should not use plugin-registry stuff there

Reply:
maybe should rename this image to `che-registry-dev`: all the required deps to build devfile or plugin registry

Reply:
well I think it's better to separate them
b/c we may want to publish che-plugin-registry-dev with different node version, etc.

Reply:
ok I inlude that to https://github.com/eclipse/che/issues/19305